test/v2/priceImpact/priceImpact.integration.test.ts

```diff
@@ -18,34 +18,50 @@ import {
     RemoveLiquidityKind,
     RemoveLiquidityUnbalancedInput,
     PoolType,
+    RemoveLiquidityNestedSingleTokenInput,
+    NestedPoolState,
+    ZERO_ADDRESS,
+    AddLiquidityNestedInput,
 } from '../../../src';
 import { ANVIL_NETWORKS, startFork } from '../../anvil/anvil-global-setup';
 import { PriceImpact } from '../../../src/entities/priceImpact';
```

Comment:
Could import from `src` to keep consistent and to make sure they are exported correctly.